test_scripts/Smoke/API/025_UnsubscribeVehicleData_PositiveCase.lua

```diff
@@ -0,0 +1,152 @@
+---------------------------------------------------------------------------------------------------
+-- User story: Smoke
+-- Use case: UnsubscribeVehicleData
+-- Item: Happy path
+--
+-- Requirement summary:
+-- [UnsubscribeVehicleData] SUCCESS: getting SUCCESS:VehicleInfo.UnsubscribeVehicleData()
+--
+-- Description:
+-- Mobile application sends valid UnsubscribeVehicleData request and gets VehicleInfo.UnsubscribeVehicleData "SUCCESS"
+-- response from HMI
+
+-- Pre-conditions:
+-- a. HMI and SDL are started
+-- b. appID is registered and activated on SDL
+-- c. appID is currently in Background, Full or Limited HMI level
+
+-- Steps:
+-- appID requests UnsubscribeVehicleData with valid parameters
+
+-- Expected:
+-- SDL validates parameters of the request
+-- SDL checks if VehicleInfo interface is available on HMI
+-- SDL checks if UnsubscribeVehicleData is allowed by Policies
+-- SDL checks if all parameters are allowed by Policies
+-- SDL transfers the VehicleInfo part of request with allowed parameters to HMI
+-- SDL receives VehicleInfo part of response from HMI with "SUCCESS" result code
+-- SDL responds with (resultCode: SUCCESS, success:true) to mobile application
+---------------------------------------------------------------------------------------------------
+
+--[[ Required Shared libraries ]]
+local runner = require('user_modules/script_runner')
+local commonSmoke = require('test_scripts/Smoke/commonSmoke')
+
+--[[ Local Variables ]]
+local VDValues = {
+  gps = "VEHICLEDATA_GPS",
+  speed = "VEHICLEDATA_SPEED",
+  rpm = "VEHICLEDATA_RPM",
+  fuelLevel = "VEHICLEDATA_FUELLEVEL",
+  fuelLevel_State = "VEHICLEDATA_FUELLEVEL_STATE",
+  instantFuelConsumption = "VEHICLEDATA_FUELCONSUMPTION",
+  externalTemperature = "VEHICLEDATA_EXTERNTEMP",
+  prndl = "VEHICLEDATA_PRNDL",
+  tirePressure = "VEHICLEDATA_TIREPRESSURE",
+  odometer = "VEHICLEDATA_ODOMETER",
+  beltStatus = "VEHICLEDATA_BELTSTATUS",
+  bodyInformation = "VEHICLEDATA_BODYINFO",
+  deviceStatus = "VEHICLEDATA_DEVICESTATUS",
+  driverBraking = "VEHICLEDATA_BRAKING",
+  wiperStatus = "VEHICLEDATA_WIPERSTATUS",
+  headLampStatus = "VEHICLEDATA_HEADLAMPSTATUS",
+  engineTorque = "VEHICLEDATA_ENGINETORQUE",
+  accPedalPosition = "VEHICLEDATA_ACCPEDAL",
+  steeringWheelAngle = "VEHICLEDATA_STEERINGWHEEL",
+  eCallInfo = "VEHICLEDATA_ECALLINFO",
+  airbagStatus = "VEHICLEDATA_AIRBAGSTATUS",
+  emergencyEvent = "VEHICLEDATA_EMERGENCYEVENT",
+  clusterModeStatus = "VEHICLEDATA_CLUSTERMODESTATUS",
+  myKey = "VEHICLEDATA_MYKEY"
+}
+
+local paramsList = {}
+local requestParams = { }
+for k, _ in pairs(VDValues) do
```

Comment:
@GetmanetsIrina propose to move this loop to `setVDRequest` local function and call it from `subscribeVD` as you did for `responseUiParams`